abstractions/go/request_adapter.go

```diff
@@ -12,18 +12,21 @@ import (
 	s "github.com/microsoft/kiota/abstractions/go/serialization"
 )
 
+// ErrorMappings is a mapping of status codes to error types factories.
+type ErrorMappings map[string]s.ParsableFactory
```

Comment:
Are clients of this package going to use `ErrorMappings`? Otherwise, consider making it private.

Reply:
The http package needs access to this type